packages/parser/src/index.ts

```diff
@@ -1,16 +1,14 @@
-import type { APIDocument, ParserOptions } from './types.js';
+import type { APIDocument, ParserOptions, ValidationResult, ErrorDetails, WarningDetails } from './types.js';
 
-import $RefParser, { dereferenceInternal } from '@apidevtools/json-schema-ref-parser';
+import $RefParser, { dereferenceInternal, MissingPointerError } from '@apidevtools/json-schema-ref-parser';
 
-import { ValidationError } from './errors.js';
 import { isSwagger, isOpenAPI } from './lib/index.js';
 import { convertOptionsForParser, normalizeArguments, repairSchema } from './util.js';
 import { validateSchema } from './validators/schema.js';
 import { validateSpec } from './validators/spec.js';
 
-export type { ParserOptions };
-
-export { ValidationError };
+export type { ParserOptions, ValidationResult, ErrorDetails, WarningDetails };
+export { ValidationError } from './errors.js';
```

Comment:
this doesn't look used now, can we delete it?

Reply:
we don't actually use it ourselves though and it's basically just the `Error` object with a different name

Reply:
i'm worried it might mislead folks into thinking it's a type that these functions actually return

Reply:
`oas-normalize` uses it. should i just move it to that library instead?

Reply:
yeah ill move it

Reply:
ohhh i see. yeah i'd prefer that, ty

Reply:
moved in 564b4a09